boards/starfive/visionfive2_jh7110/doc/index.rst

```diff
@@ -0,0 +1,74 @@
+.. _visionfive2_jh7110:
+
+Starfive VisionFive2 JH7110
+###########################
+
+Overview
+********
+
+Starfive VisionFive 2 is the world’s first high-performance RISC-V single board computer (SBC) with an integrated GPU. Compared with its last generation, VisionFive 2 has been fully upgraded with significant improvements in the processor work frequency, multimedia processing capabilities, scalability, etc. Its superior performance and reasonable price make VisionFive 2 the best affordable RISC-V development board ever.
+
+.. figure:: img/Visionfive2.jpg
+   :align: center
+   :alt:
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``visionfive2_jh7110`` board configuration can be built
+as usual (see :ref:`build_an_application`) using the corresponding board name:
+
+.. zephyr-app-commands::
+   :board: visionfive2_jh7110
+   :goals: build
```

Comment:
the soc is jh7110? So the board should be `visionfive2` then if there is only a single soc, that is the name that is used to build, if there is multiple socs then it would be built using e.g. `visionfive2/jh7110`

Reply:
Modified as per this suggestion. The board name is now "visionfive2", updated the documentation according to that